frontend/src/scenes/trends/Trends.js

```diff
@@ -66,42 +65,7 @@ export function Trends() {
                         Graph
                         <div className="float-right">
```

Comment:
![image](https://user-images.githubusercontent.com/1727427/79110355-2644ac80-7d72-11ea-9790-50a8d439c2f6.png)
These are all a bit wide, even if the text in them isn't very wide. I also personally preferred the slightly more subtle look we had before where it was sunk in and just looked like a word, but I'm happy to leave as is if you think it's better

Reply:
agreed! they do look a too port-holey right now. i made some tweaks

Reply:
Looks good now 👍